### PR TITLE
Refactored "Projects" into "Sessions"

### PR DESCRIPTION
**Pull request by NelsonPereira1991**
Refactored Projects into sessions, also refactored on the frontend side(calls to the api).

Guys please be carefull when reviewing this. I tested the changes but take a closer look

### CODE REVIEW

File: src/middleware/measurement.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const path = require("path");
 const modelsFolder = global.modelsFolder;
 const measurementModel = require(path.resolve(modelsFolder, "measurement")).Measurement;
 const responseUtil = require(path.resolve(global.utilsFolder, "response"));
-const iterationModel = require(path.resolve(modelsFolder, "iteration")).Project;
```
Comment (jorgemiguelreis) [inline src/middleware/measurement.js]: It was wrong right? Is it needed?

File: src/models/session.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        min: 1,
+        max: 8
+    },
+    numRuns: {
+        type: Number,
+        required: true,
+        min: 1,
+        max: 10
+    },
+    timePerRun: {//minutes??
```
Comment (jorgemiguelreis) [inline src/models/session.js]: yes, minutes :)

File: src/middleware/measurement.js (hunk 1 above)
Comment (NelsonPereira1991) [reply src/middleware/measurement.js]: yes, not needed anymore

Comment (NelsonPereira1991) [reply src/middleware/measurement.js]: I removed it

**Pull request merged**